2022/Preliminary/1 Chantu Bantu/Solution.kt

```diff
@@ -0,0 +1,25 @@
+data class TestCase(
```

Comment:
@shannon-vaz  could you please add a comment stating the score - I believe you must have scored 100/100. This is to help the reader to understand whether this is a complete soln or a partial one. Thanks.

Reply:
Added